orchestrator/types.py

```diff
@@ -186,7 +215,7 @@ def is_optional_type(t: Any, test_type: Optional[type] = None) -> bool:
     False
     """
     if get_origin(t):
```

Comment:
Is this check needed, since on the next line we compare it with Union

Reply:
I'm afraid we need the test:

```python
>>> h = "hoi"
>>> g = Union[str,int]
>>> x = get_origin(h)
>>> x
>>> x = get_origin(g)
>>> x
typing.Union
```

Reply:
I know, but on the next there is a test with 'if get_origin(t) == Union' . I would assume that test would also fail if get_origin(t) returns a falsey value.